src/day4/Vehicle.java

```diff
@@ -0,0 +1,69 @@
+package day4;
+
+public class Vehicle {
+    final static double DEFAULT_FUEL_CONSUMPTION = 1.25;
+    private double fuelConsumption;
+    private double fuel;
+    private int horsePower;
+
+    public Vehicle(double fuel, int horsePower) {
+        this.fuel = fuel;
+        this.horsePower = horsePower;
+        this.fuelConsumption = DEFAULT_FUEL_CONSUMPTION;
+    }
+
+    public double getFuelConsumption() {
+        return this.fuelConsumption;
+    }
+    public void setFuelConsumption(double fuelConsumption) {
+        if (fuelConsumption <= 0) {
+            throw new IllegalArgumentException();
+        }
```

Comment:
Can put the else on the same line as the closing brackets. https://www.w3schools.com/java/java_conditions.asp